### PR TITLE
webui rename files without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUIRenameFiles``` to speed up CI
## Related Issue

Part of https://github.com/owncloud/QA/issues/622

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/webUIRenameFiles/renameFiles.feature [hunk 1]
```diff
@@ -70,9 +79,11 @@ Feature: rename files
     Then file "  multiple   space    all     over   .  dat  " should be listed on the webUI
 
   Scenario: Rename a file using both double and single quotes
+    Given user "user1" has uploaded file with content "some content" to "/randomfile.txt"
+    And the user reloads the current page of the webUI
```
Comment (dpakach) [inline tests/acceptance/features/webUIRenameFiles/renameFiles.feature]:
```suggestion
    And the user has reloaded the current page of the webUI
```

File: tests/acceptance/features/webUIRenameFiles/renameFiles.feature [hunk 2]
```diff
@@ -5,23 +5,27 @@ Feature: rename files
...
-    Given user "user1" has been created with default attributes and skeleton files
+    Given user "user1" has been created with default attributes and without skeleton files
     And user "user1" has logged in using the webUI
     And the user has browsed to the files page
 
   @smokeTest
   Scenario Outline: Rename a file using special characters
-    When the user renames file "lorem.txt" to <to_file_name> using the webUI
+    Given user "user1" has uploaded file with content "some content" to "/randomfile.txt"
+    And the user reloads the current page of the webUI
```
Comment (dpakach) [inline tests/acceptance/features/webUIRenameFiles/renameFiles.feature]:
```suggestion
    And the user has reloaded the current page of the webUI
```
also in other scenarios

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by individual-it — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
Looks good.
Most scenarios upload `randomfile.txt`. Unfortunately not all, so it cannot go in the Background section.

**Comment by HariBhandari07** [comment]
Backport on #35495

**Pull request merged**